game/core/src/me/lihq/game/screen/NavigationScreen.java

```diff
@@ -229,11 +235,17 @@ public void finishRoomTransition()
     @Override
     public void render(float delta)
     {
-
         game.player.pushCoordinatesToSprite();
+        for (AbstractPerson n : currentNPCS) {
+            n.pushCoordinatesToSprite();
+
+        }
 
         if (changeMap) {
             tiledMapRenderer.setMap(game.player.getRoom().getTiledMap());
+            tiledMapRenderer.clearPeople();
+            tiledMapRenderer.addPerson((List<AbstractPerson>) ((List<? extends AbstractPerson>) currentNPCS));
```

Comment:
maybe, if not rename to getNPC -> getPeople although then it's not accurate as players are also people but they wont be in the getPeople list